tests/phpunit/src/Commands/Api/ApiCommandTest.php

```diff
@@ -384,7 +392,7 @@ public function testConvertInvalidEnvironmentAliasToUuidArgument(): void
         $this->command = $this->getApiCommandByName('api:environments:find');
         $alias = 'devcloud2.invalid';
         $this->expectException(AcquiaCliException::class);
-        $this->expectExceptionMessage('{environmentId} must be a valid UUID or site alias.');
+        $this->expectExceptionMessage('Environment not found matching the alias devcloud2.invalid');
```

Comment:
This is a friendlier exception, I'd prefer to allow validateEnvironmentUuid to propagate it rather than catching and throwing something custom.